StandaloneFirefoxDebug/Dockerfile

```diff
@@ -7,6 +7,11 @@ LABEL authors=SeleniumHQ
 
 USER seluser
 
+#====================================
```

Comment:
Did you check this [comment](https://github.com/SeleniumHQ/docker-selenium/blob/master/StandaloneChromeDebug/Dockerfile#L2)?

Please modify that file first.